Update group_by.py

Vebryn opened the pull request:
add note related to when issue

Vebryn committed:
Update group_by.py

add note related to when issue

bcoca commented:
just a link is not very helpful, also it is not easy to understand the cause from the ticket itself w/o an explanation

Vebryn committed:
Update group_by.py

two lines in order to better understand github issue

Vebryn commented:
how to merge commits to ansible doc ?

gregdek commented:
Thanks @Vebryn. @jhoekx please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

robynbergeron commented:
@jhoekx This change is still pending your review; do you have time to take a look and comment? Please comment with text 'shipit' or 'needs_revision' as appropriate.

jhoekx commented on inventory/group_by.py:
```diff
@@ -30,6 +30,9 @@
 author: "Jeroen Hoekx (@jhoekx)"
 notes:
   - Spaces in group names are converted to dashes '-'.
+  - Use of 'when' directive cause unpredictable results.
```
I'm not a native speaker, but I think the message should be improved.

Something like: 
- Use of group_by with the when statement causes unpredictable results.
- Only variables of the first host in the inventory are available.

jhoekx commented:
needs_revision

gregdek commented:
Thanks @Vebryn for this PR. A maintainer of this module has asked for revisions to this PR. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

Vebryn committed:
Update group_by.py

Hope it's better. I also not a native speaker. I think my sentences are correct. Is there a native speaker here ?

Best regards.

gregdek commented:
@Vebryn A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

Vebryn commented:
Action done with commit [310c1b9](https://github.com/ansible/ansible-modules-core/pull/2397/commits/310c1b9f857a922ca92685653b163a96b0121803), any other action needed ?

gregdek commented:
@Vebryn Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

Vebryn commented:
ready_for_review

gregdek commented:
Thanks @Vebryn. To the current maintainers, @jhoekx please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit' or 'needs_revision' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

gregdek commented:
Thanks @Vebryn for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]
